x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -335,15 +336,31 @@ export class FleetPlugin
     });
     licenseService.start(this.licensing$);
 
-    const fleetServerSetup = startFleetServerSetup();
-
     this.telemetryEventsSender.start(plugins.telemetry, core);
 
+    const logger = appContextService.getLogger();
+
+    const fleetSetupPromise = startFleetServerSetup()
+      .then(() => {
+        logger.info('Beginning fleet setup');
+
+        return setupFleet(
```

Comment:
This function return `nonFatalErrors` should we log them? otherwise the user will have no idea of these errors